client/src/helpers/exportData.js

```diff
@@ -0,0 +1,53 @@
+// For exporting the table data
+import { ExportToCsv } from 'export-to-csv'; //or use your library of choice here
+const  getDate = ()=> {
+    const today = new Date();
+    const year = today.getFullYear();
+    const month = String(today.getMonth() + 1).padStart(2, '0');
+    const day = String(today.getDate()).padStart(2, '0');
+    const formattedDate = `${year}-${month}-${day}`;
+    return formattedDate
+  }
```

Comment:
This function was created but not used in other files, so we have duplicated code

Reply:
removed duplicates made an import for it